.github/workflows/build.yml

```diff
@@ -24,8 +24,21 @@ jobs:
           check-latest: true
       - name: Build CLI
         run: make build-linux-amd64
-      - name: Run Unit Tests
-        run: make test
+      - name: Start Distribution
+        run: |
+          docker stop oras-e2e || true
+          docker run -d -p 5000:5000 --rm --name oras-e2e \
+            --mount type=bind,source=$GITHUB_WORKSPACE/test/e2e/testdata/config-example-with-extensions.yml,target=/etc/docker/registry/config.yml \
+            --mount type=bind,source=$GITHUB_WORKSPACE/test/e2e/testdata/passwd_bcrypt,target=/etc/docker/registry/passwd \
+            ghcr.io/oras-project/registry:latest
+      - name: Run Tests
+        run: |
+          trap docker stop oras-e2e || true ERR
+          make test
+          docker stop oras-e2e || true
```

Comment:
What's the difference between `docker stop` and `docker kill`?

Reply:
looks like `docker stop` will wait 10s and then kill? Should use kill to save time.